webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Move mode flag here:
```suggestion
module.exports = {
  mode: 'production',
```